src/main/java/frc/robot/commands/shooter/AutoScore.java

```diff
@@ -23,31 +35,84 @@ public AutoScore(
       double AmpPercentOutput,
       double RPMOutput,
       double FrontIntakeAmpPercentOutput,
-      Double BackIntakeAmpPercentOutput) {
+      Double BackIntakeAmpPercentOutput,
+      double TimeToShoot,
+      double withTimeout) {
     m_shooter = shooter;
     m_ampShooter = ampShooter;
     m_intake = intake;
     m_AmpPercentOutput = AmpPercentOutput;
     m_FrontIntakePercentOutput = FrontIntakeAmpPercentOutput;
     m_BackIntakeAmpPercentOutput = BackIntakeAmpPercentOutput;
     m_RPMOutput = RPMOutput;
+    m_timeToShoot = TimeToShoot;
+    m_withTimeout = withTimeout;
 
     addRequirements(m_shooter);
   }
 
   // Called when the command is initially scheduled.
   @Override
-  public void initialize() {}
+  public void initialize() {
+    m_timer.stop();
+    m_timer.reset();
+    m_reversetimer.stop();
```

Comment:
I'm not familiar with timer logic, but I'll trust that this is correct on initialize?